workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -364,6 +375,7 @@ def _step_prephysics(self) -> Diagnostics:
             "override_for_time_adjusted_total_sky_downward_shortwave_flux_at_surface",
             "override_for_time_adjusted_total_sky_net_shortwave_flux_at_surface",
             "override_for_time_adjusted_total_sky_downward_longwave_flux_at_surface",
+            "ocean_surface_temperature",
```

Comment:
@brianhenn  I think prephysics is the appropriate step for the SST update so I put this here. However,  it's possible to have a case where you provide a variable in the `prephysics` through a dataset and there is no postphysics stepper, in which case the run will proceed without prescribing from the specified data. Do we want to raise some exception for those cases and have a separate configuration for prescribing state postphysics ? Rename the config section to something more general like `prescribe_state`, rely on the `prephysics_overrides` list to determine where the updates happen, and let the state update in the postphysics step regardless of whether there's a nudging or ML stepper?

Reply:
Yeah, good point. Being able to configure where state updates get applied seems like too much flexibility though, and this is a pretty short list, so I'd be more in favor of just raising an exception if someone tries to prescribe a variable that isn't on this list and there's no postphysics stepper.

This seems like an ongoing challenge that there's not really documentation of when the different state and tendency updates will occur and under what conditions, outside of just staring at the loop for a while. Maybe it'd be helpful to at least list the allowed `prephysics_overrides` in the `prephysics` UserConfig docs and say that any other variables will only be set postphysics if there's a stepper, and that the only variable we've really tried doing this with is `total_precipitation`. I know that it'll get stale in the docstring but at least it's somewhere.

Reply:
There's an old issue that's also related to this discussion (https://github.com/ai2cm/fv3net/issues/1211). I agree that we shouldn't use the config to decide where the updates happen. As per our in person discussion just now I'm going to 

i) rename the `prephysics` part of the config to `prescribed_state`, since it also can add postphysics updates
ii) create another whitelist of `postphysics_overrides` that will get applied in the postphysics step, raise an error if there are variables in the prescribed state config that are in neither override list. This means any new override variables require a PR to update one of these lists.
iii) apply postphysics overrides regardless of whether there is a postphysics stepper